javascript/central-login-oidc/src/main.js

```diff
@@ -43,8 +43,14 @@ const showUser = (user) => {
 
 const logout = async () => {
   try {
-    await forgerock.FRUser.logout();
-    location.assign(`${document.location.origin}/`);
+    if (process.env.SERVER_TYPE === "PINGONE") {
```

Comment:
I wonder if a "toLowerCase" on these strings would cut down on possible errors?
I know we say the value _must_ be either "PINGONE" or "AIC", but I bet there will be people who don't case the string as we'd expect.

Reply:
I prefer uppercase if it's not a BOOL. As these are Env settings and should make people think and read it clearly. It's not uncommon practice I think